Community/Tdarr_Plugin_goof1_URL_Plex_Refresh.js

```diff
@@ -186,7 +186,8 @@ const plugin = async (file, librarySettings, inputs, otherArguments) => {
   response.infoLog += `Attempting to update Plex path ${filePath} in library ${key}\n`;
 
   const portIfUsed = port ? `:${port}` : '';
-  const urlNoToken = `${type}://${url}${portIfUsed}/library/sections/${key}/refresh?path=${filePath}&X-Plex-Token=`;
+  const urlNoToken = `${type}://${url}${portIfUsed}/library/sections/${key}/refresh?\
+path=${encodeURIComponent(filePath)}&X-Plex-Token=`;
```

Comment:
It'll end up with a new line in the URL this way, please terminate the string and concat them.

Reply:
I no longer have access to a computer to make the changes and won't have one for several weeks while mine gets repaired so feel free to push directly to the pr.

but are you sure? Since there is no white spaces between the backslash and "path" it should end up in a new line as the backspace is escaping the new line.

Reply:
Didn't notice the backslash, but it's a little neater to do it the way I described. I'll do it once I'm near a pc.